docs/overview/policy-lifecycle.mdx

```diff
@@ -82,20 +82,52 @@ git tag v0.1.2 && git push --tags
 
 ---
 
-**GitHub Actions Secret**
+**Manual Setup for GitHub Actions**
 
-Aserto can set up a secret for you when you create a new policy.
-You can also setup this secret yourself, using information from the **Policy Settings** tab in the Aserto Console.
+In order to set up GitHub Actions manually, use the `policy` CLI and run the following command to initialize a new policy repository:
 
-The secret should be named `ASERTO_PUSH_KEY` and it should contain a JSON object that looks like this:
+```bash
+policy templates apply github-policy-ci
+```
+
+Initially you'll have to select the policy repository you'd like to use:
 
-```json
-{
-  "tenant_id": "",
-  "policy_id": "",
-  "push_key": ""
-}
+```bash
+  #  SERVER
+  1  opcr.io
+  2  registry.prod.aserto.com
+
+> Select server#:
+```
+Select the `registry.prod.aserto.com` server. Next, provide your GitHub username, for example:
+
+```bash
+> user (): rschwabco
+```
+Next, you'll be asked to provide the _name_ of the GitHub secret that will be used to push the policy image to the registry.
+
+```bash
+> secret name (TOKEN):
 ```
+You can select the default secret name ("TOKEN") or change it to a different name if you so choose. Note that you'll need to use this secret name when you create the secret in your GitHub repository.
+
+Finally, you'll be asked to provide the GitHub organization and repository you'll be using. For example:
+
+```bash
+> org/repo: rschwabco/policy-test
+```
+
+Once completed, the CLI will generate a `.github` folder which will include the workflow file required to build and push policy changes to the Aserto Container Registry whenever a new tag is pushed to the GitHub repository.
+
+To complete the setup, head to the Aserto console and retrieve the APCR API key. Head to the Connections tab, and click on "Aserto Policy Registry".
```

Comment:
Instead of APCR API key, I think you should spell it out "Aserto Policy Registry API key".